### PR TITLE
nikolai.berestevich/refactor/delete  alt tags for icons /markets 39155

### PR DESCRIPTION
**Pull request by nikolaiberestevich-deriv**
Changes:

-   delete  alt tags for icons /markets 39155

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ x ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/landing/index.js [hunk 1]
```diff
@@ -17,21 +17,21 @@ const simple_step_content = [
         text: (
             <Localize translate_text="Open a demo account and start trading for free. Practise with an unlimited amount of virtual funds." />
         ),
-        icon: <img src={PractiseIcon} alt="aim" width="32" height="32" />,
+        icon: <img src={PractiseIcon} width="32" height="32" />,
```
Comment (sean-binary) [inline src/pages/landing/index.js:20]: We need empty alt tags for these. We can't remove it entirely.

Comment (njazuli-deriv) [reply src/pages/landing/index.js:20]: I agree on this.. https://web.dev/image-alt/ for reference

Comment (nikolaiberestevich-deriv) [reply src/pages/landing/index.js]: 👌

Comment (nikolaiberestevich-deriv) [reply src/pages/landing/index.js]: done

File: src/pages/markets/components/helper/_available-platforms.js [hunk 2]
```diff
@@ -64,12 +64,12 @@ const AvailablePlatforms = ({
     derivx,
     flex_direction,
     tablet_direction,
-    }) => {
+}) => {
     const { is_eu_country } = React.useContext(DerivStore)
 
     return (
         <Flex
-            style={{'flexWrap': 'wrap'}}
```
Comment (sean-binary) [inline src/pages/markets/components/helper/_available-platforms.js:72]: I know this isn't the scope but just can't help noticing this. Let's remove this and use the `wrap` prop instead to pass in the wrap property for `Flex`.

**Review by sean-binary — DISMISSED** [review]
Rest LGTM

**Review by sean-binary — APPROVED** [review]

File: src/pages/markets/components/helper/_available-platforms.js (hunk 2 above)
Comment (nikolaiberestevich-deriv) [reply src/pages/markets/components/helper/_available-platforms.js:72]: > I know this isn't the scope but just can't help noticing this. Let's remove this and use the `wrap` prop instead to pass in the wrap property for `Flex`.

OK => done in the 4th commit =)

**Pull request merged**